### ISSUE TITLE
libvidstab.so.1.1: cannot open shared object file

### ISSUE THREAD

**Issue by wesbluemarine** [skycocker/chromebrew#2442]
## Description
**FFmpeg** does not find **vid.stab** that is installed already.

```shell
$ ffmpeg
ffmpeg: error while loading shared libraries: libvidstab.so.1.1: cannot open shared object file: No such file or directory
```
```shell
$ crew search vidstab
(i) vidstab: Transcode video stabilization plugin.
```

## Versions and system information
chromebrew: `0.4.3`

ChromeOS: `$ cat /etc/lsb-release`

`CHROMEOS_RELEASE_BOARD=chromeover64
CHROMEOS_DEVSERVER=https://cloudready-free-update-server.neverware.com/
GOOGLE_RELEASE=66.4.25
CHROMEOS_RELEASE_BUILD_NUMBER=10452
CHROMEOS_RELEASE_BRANCH_NUMBER=77
CHROMEOS_RELEASE_CHROME_MILESTONE=66
CHROMEOS_RELEASE_PATCH_NUMBER=2018_06_25_1925
CHROMEOS_RELEASE_TRACK=developer-build
CHROMEOS_RELEASE_DESCRIPTION=66.4.25 (Developer Build - neverware) developer-build chromeover64
CHROMEOS_RELEASE_NAME=Chromium OS
CHROMEOS_RELEASE_BUILD_TYPE=Developer Build - neverware
CHROMEOS_RELEASE_VERSION=66.4.25
CHROMEOS_AUSERVER=https://cloudready-free-update-server.neverware.com/update`

Model: Cloudready Custom PC

Architecture: `x86_64`

## Addtional information

I think it's missing some **symlink**.

---

**Comment by Axloqsizlik** [comment]
I ran into this too.

**Comment by JL2210** [comment]
Could you give me the output of `crew const`?

**Comment by wesbluemarine** [comment]
`ARCH=x86_64
ARCH_LIB=lib64
CHROMEOS_RELEASE=66
CREW_BREW_DIR=/usr/local/tmp/crew/
CREW_CONFIG_PATH=/usr/local/etc/crew/
CREW_DEST_DIR=/usr/local/tmp/crew/dest
CREW_DEST_LIB_PREFIX=/usr/local/tmp/crew/dest/usr/local/lib64
CREW_DEST_PREFIX=/usr/local/tmp/crew/dest/usr/local
CREW_LIB_PATH=/usr/local/lib/crew/
CREW_LIB_PREFIX=/usr/local/lib64
CREW_NOT_COMPRESS=
CREW_NOT_STRIP=
CREW_NPROC=8
CREW_PREFIX=/usr/local
CREW_VERSION=0.4.3
USER=chronos`

**Comment by JL2210** [comment]
Try powerwashing and not enabling debug info.

**Comment by Axloqsizlik** [comment]
LD_LIBRARY_PATH=/usr/local/lib:/usr/local/lib64:$LD_LIBRARY_PATH && export LD_LIBRARY_PATH
maybe that works?

**Comment by dannyhimself** [comment]
`LD_LIBRARY_PATH=/usr/local/lib:/usr/local/lib64 && export LD_LIBRARY_PATH`

This code from 00fer0n1 also fixed the issue for me, however i seemed to be missing libomp.so which i installed through llvm with chromebrew
`crew install llvm`

As suggested by wesbluemarine the first commad should be included in .bashrc to make things easier and stop the need to set this in each new shell.

**Comment by uberhacker** [comment]
Fixed in #2855.
